spec/serializers/api/unfunded_mentor_serializer_spec.rb

```diff
@@ -0,0 +1,51 @@
+describe API::UnfundedMentorSerializer, type: :serializer do
+  subject(:response) do
+    options = { lead_provider_id: lead_provider.id }
+    JSON.parse(described_class.render(unfunded_mentor_teacher, **options))
+  end
+
+  let!(:lead_provider) { FactoryBot.create(:lead_provider) }
+  let(:created_at) { Time.utc(2023, 7, 1, 12, 0, 0) }
+  let(:api_updated_at) { Time.utc(2023, 7, 2, 12, 0, 0) }
+  let(:unfunded_mentor_teacher) do
+    FactoryBot.create(
+      :teacher,
+      created_at:,
+      api_updated_at:
+    )
+  end
+
+  before do
+    # Create mentor at school periods for the unfunded mentor teacher
+    FactoryBot.create(:mentor_at_school_period, :ongoing, teacher: unfunded_mentor_teacher, started_on: 6.months.ago, email: "test1@test.com")
+    FactoryBot.create(:mentor_at_school_period, teacher: unfunded_mentor_teacher, started_on: 1.year.ago, finished_on: 6.months.ago, email: "test2@test.com")
+
+    # Ensure other metadata exists for another lead provider.
```

Comment:
I don't think we need this as we're not actually using metadata in this serializer?